string-manipulation-2/src/capitalize-word.js

```diff
@@ -0,0 +1,24 @@
+/* exported capitalizeWord */
+
+// create variable for lowe rcase letters, upperCase letter ,result
+// use charAt() to capitalize 1 letter and store if variable
+// slice 1 letter
+// write conditional statemant
+
+function capitalizeWord(word) {
+  var lowerCaseString = '';
+  var upperCaseLetter = '';
+  var resultOfSlice = '';
+  var result = '';
+  lowerCaseString = word.toLowerCase();
+  upperCaseLetter = lowerCaseString.charAt(0).toUpperCase();
+  resultOfSlice = lowerCaseString.slice(1);
+  result = upperCaseLetter.concat(resultOfSlice);
+
+  if (lowerCaseString === 'javascript') {
+    return 'JavaScript';
```

Comment:
You should do this conditional before anything else in the function, that way you don't do a bunch of stuff that ends up being useless if the value was `"javascript"`